updates/1/updates.sql

```diff
@@ -26,3 +26,37 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xBBc2AE13b23d7
+*
+INSERT INTO assets(identifier,type,name,symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("CRON", "B", "Cryptocean", "CRON", 1557885600, NULL, "cryptocean", "CRON", "CRON"); INSERT INTO common_asset_details(asset_id, forked) VALUES("CRON", NULL);
+*
+INSERT INTO assets(identifier,type,name,symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("FIDA", "Y", "Bonfida", "FIDA", 1608598800, NULL, "bonfida", "FIDA", "FIDA"); INSERT INTO common_asset_details(asset_id, forked) VALUES("FIDA", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x50D1c9771902476076eCFc8B2A83Ad6b9355a4c9", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x50D1c9771902476076eCFc8B2A83Ad6b9355a4c9", "C", "FTX Token", "FTT", 1555852887, NULL, "ftx-token", "FTT", "0x50D1c9771902476076eCFc8B2A83Ad6b9355a4c9");
+*
+INSERT INTO assets(identifier,type,name,symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("HOLY", "Y", "Holy Trinity", "HOLY", 1609894800, NULL, "holy-trinity", "", "HOLY"); INSERT INTO common_asset_details(asset_id, forked) VALUES("HOLY", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x3E9BC21C9b189C09dF3eF1B824798658d5011937", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x3E9BC21C9b189C09dF3eF1B824798658d5011937", "C", "Linear", "LINA", 1600266838, NULL, "linear", "LINA", "0x3E9BC21C9b189C09dF3eF1B824798658d5011937");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xB1f66997A5760428D3a87D68b90BfE0aE64121cC", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xB1f66997A5760428D3a87D68b90BfE0aE64121cC", "C", "LuaSwap", "LUA", 1601024188, NULL, "lua", "LUA", "0xB1f66997A5760428D3a87D68b90BfE0aE64121cC");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x08d967bb0134F2d07f7cfb6E246680c53927DD30", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x08d967bb0134F2d07f7cfb6E246680c53927DD30", "C", "MATH", "MATH", 1569577860, NULL, "math", "MATH", "0x08d967bb0134F2d07f7cfb6E246680c53927DD30");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x965697b4ef02F0DE01384D0d4F9F782B1670c163", 6, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x965697b4ef02F0DE01384D0d4F9F782B1670c163", "C", "Oxygen", "OXY", 1607784270, NULL, "oxygen", "OXY", "0x965697b4ef02F0DE01384D0d4F9F782B1670c163");
```

Comment:
```suggestion
INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x965697b4ef02F0DE01384D0d4F9F782B1670c163", 6, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x965697b4ef02F0DE01384D0d4F9F782B1670c163", "C", "Oxygen Ecosystem Token", "OXY", 1607784270, NULL, "oxygen", "OXY", "0x965697b4ef02F0DE01384D0d4F9F782B1670c163");
```

name is `Oxygen Ecosystem Token`: https://etherscan.io/address/0x965697b4ef02F0DE01384D0d4F9F782B1670c163#readContract